src/components/Tooltip/WTooltip.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="group inline relative p-[4px] -p-[4px]">
```

Comment:
Why are we applying `padding: 4px` and `padding: -4px`?

Reply:
its a bug, it has to have negative margin to expand hover area